docs/security-policy.md

```diff
@@ -13,17 +13,16 @@ and Pods are attached to NSX-T segments, each workload has segment port on NSX-T
 Labels on VMs and Pods are tagged on their segment ports.
 In the first version, nsx-operator leverages Tags added by NSX Container Plugin(NCP)
 on NSX-T segment ports, nsx-operator needs to work together with NCP on Supervisor
-cluster of vSphere with Kubernetes. 
+cluster of vSphere with Kubernetes.
 nsx-operator will reconcile SecurityPolicy CRs, call NSX-T API to create
 NSX-T Distributed Firewall rules, then update CR status with realized state.
 
 ## CRD Design
 
-SecurityPolicy is Namespaced scope CRD, admins could apply CR within a namespace to
-configure security of workloads which are attached to NSX-T networking.
-Compared with standard K8s NetworkPolicies, SecurityPolicy CRD adds `VMSelector` to
-select VMs, it also introduces some rule syntaxes including Drop and Reject actions,
-priority, rule level selectors(`appliedTo`). 
+SecurityPolicy is Namespaced scope CRD, admins could apply CR within a namespace to configure security of workloads
+which are attached to NSX-T networking. Compared with standard K8s NetworkPolicies, SecurityPolicy CRD adds `vmSelector`
+to select VMs, it also introduces some rule syntax including Drop and Reject actions, priority, rule level
+selectors(`appliedTo`).
 
 An example of SecurityPolicy CR:
```

Comment:
It's better to add a named port in this example since we want to cover usage of SecurityPolicy as much as possible in this one, including matchLables, matchExpressions, namespaceSelector, vmSelector,podSelector and named port etc. It will be a good reference for users.